packages/fxa-settings/src/pages/InlineTotpSetup/container.tsx

```diff
@@ -178,10 +180,16 @@ export const InlineTotpSetupContainer = ({
     return <LoadingSpinner fullScreen />;
   }
 
+  const isUpdatedInlineTotpSetupFlow =
+    config.featureFlags?.updatedInlineTotpSetupFlow || false;
+
+  const InlineTotpSetup = isUpdatedInlineTotpSetupFlow
+    ? InlineTotpSetupNew
+    : InlineTotpSetupOld;
+
   return (
     <InlineTotpSetup
       {...{ totp, serviceName, cancelSetupHandler, verifyCodeHandler }}
```

Comment:
Alternate suggestion, to avoid passing in unnecessary props:

```suggestion

return (
   {isUpdatedInlineTotpSetupFlow ? 
     <InlineTotpSetupNew {...{totp, serviceName, verifyCodeHandler}} /> :
     <InlineTotpSetupOld {...{totp, serviceName, cancelSetupHandler, verifyCodeHandler}} />
   }
```